packages/cli/src/run.js

```diff
@@ -28,7 +29,7 @@ export default async function run (_argv, hooks = {}) {
   let cmd = await getCommand(argv[0])
 
   // Matching `nuxt` or `nuxt [dir]` or `nuxt -*` for `nuxt dev` shortcut
-  if (!cmd && (!argv[0] || argv[0][0] === '-' || (argv[0] !== 'static' && fs.existsSync(argv[0])))) {
+  if (!cmd && (!argv[0] || argv[0][0] === '-' || fs.existsSync(`${argv[0]}${path.sep}nuxt.config.js`) || fs.existsSync(`${argv[0]}${path.sep}pages`) || fs.existsSync(`${argv[0]}${path.sep}nuxt.config.ts`))) {
```

Comment:
@pi0 What do you think about the order of these 3. Should we put `pages` at first?

Reply:
It is fine just would be better if we refactor out into a util function :D

Reply:
Yeah, I just want to confirm it.
I was thinking about creating `isNuxtDir` or something like it. It will be useful for modules, especially in the Storybook.
https://github.com/nuxt-community/storybook/blob/master/src/cli.ts#L27

Reply:
Maybe we cannot reuse as it would require using latest nuxt for storybook but can use same logic :)

Reply:
Isn't it a feature? 😅